projects/tad/index.md

```diff
@@ -0,0 +1,84 @@
+|      | Functionality          | The tool does not share any data or logging to anyone or anywhere. | C        |
+|      | Functionality          | The tool can be integrated in a CI/CD flow.                  | C        |
+|      | Functionality          | The tool can be offered as a (cloud) service where no local installation is required. | C        |
+|      | Reliability            | The tool can operate consistently and reliably.              | M        |
+|      | Reliability            | The tool can recover automatically from common failures.     | M        |
+|      | Reliability            | The tool recovers from failures quickly and minimizes data loss. | S        |
+|      | Reliability            | The tool handles errors gracefully and informs users of any issues. | S        |
+|      | Reliability            | The tool provides clear error messages and instructions for troubleshooting. | S        |
+|      | Usability              | The tool possess a clean, intuitive, and visually appealing UI that follows industry standards. | S        |
+|      | Usability              | The tool offers keyboard shortcuts for efficient interaction. | C        |
+|      | Usability              | The tool provides clear and consistent navigation, making it easy for users to find what they need. | S        |
+|      | Usability              | The tool is responsive and provides instant feedback.        | S        |
+|      | Usability              | The user interface is multilingual and supports at least English. | S        |
+|      | Usability              | The user interface can easily be translated into other languages. | C        |
+|      | Help & Documentation   | The tool provides comprehensive online help documentation with searchable functionalities. | S        |
+|      | Help & Documentation   | The tool offers context-sensitive help within the application. | C        |
+|      | Help & Documentation   | The online documentation includes video tutorials and training materials for ease of learning. | C        |
+|      | Help & Documentation   | The project provides readily available customer support through various channels  (e.g., email, phone, online chat) to address user inquiries and troubleshoot issues. | C        |
+|      | Performance Efficiency | The tool operates efficiently and minimize resource utilization. | M        |
+|      | Performance Efficiency | The tool responds to user actions instantly.                 | M        |
+|      | Performance Efficiency | Resource consumption of the tool itself (e.g., CPU, memory) does not exceed [Threshold] under normal load. Resource consumption of the tests can not be predicted. | C        |
+|      | Performance Efficiency | The tool is scalable to accommodate increased user base and data volume. | S        |
+|      | Maintainability        | The project is open source.                                  | S        |
```

Comment:
Although i would agree with S, maybe M would be better as it is one of our core principles to prefer open-source https://minbzk.github.io/ai-validation/Way-of-Working/Principles/